payment-service/src/main/resources/application.properties

```diff
@@ -3,12 +3,12 @@ server.port=0
 
 
 eureka.client.service-url.defaultZone=http://localhost:8761/eureka/
-spring.jpa.hibernate.ddl-auto=none
-spring.datasource.url=jdbc:mysql://localhost:3306/libary
-spring.datasource.username=root
+spring.jpa.hibernate.ddl-auto=create
+spring.datasource.url=${USERMANAGER_DATABASE_URL}
+spring.datasource.username=${USERMANAGER_DATABASE_USERNAME}
 spring.datasource.password=${USERMANAGER_DATABASE_PASSWORD}
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.liquibase.change-log=classpath:db/changelog/db.changelog-master.xml
+spring.liquibase.change-log=classpath:db/changelog/db.changelog_master.xml
```

Comment:
Also it would be nice to add numbers to changelogs or months for changelog files (except master one):
2023.11_initial_scripts.xml

Reply:
renamed this.